site/docs/concepts/architecture.md

```diff
@@ -3,31 +3,26 @@
 
-Once a developer submits a `FybrikApplication` CRD to Kubernetes, the FybrikApplicationController will make sure that all the specs are fulfilled and will make sure that the data is read/written/copied/deleted according to the data governance policies. The `FybrikApplication` holds metadata about the application such as the data assets required by the application, the processing purpose and the method of access the user wishes (protocol e.g. S3 or Arrow flight). 
-It uses this information to check with the data governance policy manager (4) if the data flow requested is allowed
-and whether restrictive policies such as masking or hashing have to be applied. It compiles plotters based on the governance decisions received via the data governance policy connector and chooses the modules (5) which are best fit for the requirements that the user specified regarding the access protocol and availability, and based on the [config policies](./config-policies.md) defined.  The plotter contains the modules required and all the information required to deploy them on the cluster where they will run, as well as the flow of data between the asset and the workload through the chosen modules.  
+Once a developer submits a `FybrikApplication` CRD to Kubernetes, the FybrikApplicationController will make sure that all the specs are fulfilled and that the data is read/written/copied/deleted in accord with the data governance policies and the IT config policies. The `FybrikApplication` holds metadata about the application such as the data assets required by the application, the processing purpose and the method of access the user wishes (protocol e.g. S3 or Arrow flight). 
+The controller uses this information to check with the data-governance policy manager if the data flow requested is allowed
+and whether restrictive actions such as masking or hashing have to be applied. Taking into account these governance actions, as well as application requirements, dataset specification, available infrastructure (5) and the [IT config policies](./config-policies.md) defined (4), the controller compiles a plotter (9). The plotter specifies a data plane connecting the application to the datasets it requires. More specifically, the plotter lists the modules to use, the capabilities required from these modules, the cluster on which each module should be deployed, as well as the flow of data between the asset and the workload through the chosen modules. For the plotter to be optimal in terms of the defined optimization goals (a.k.a. IT config soft policies), the controller may use a [CSP-based optimizer](./optimizer.md) (7).  If no CSP engine is installed, soft policies will not be taken into account, and the manager will use the first (but not necessarily optimal) solution that meets all of the other requirements.
```

Comment:
We decided to use the term "optimization goals" rather than "soft policies"

Reply:
@simanadler do you agree?

Reply:
@zivnevo Yes, optimization policies is what we decided

Reply:
fixed